glue/sales_val_flagging.py

```diff
@@ -562,7 +553,7 @@ def write_to_table(df, table_name, s3_warehouse_bucket_path, run_id):
         WHERE condo.class IN ('297', '299', '399')
         AND NOT condo.is_parking_space
         AND NOT condo.is_common_area
-        AND condo.is_question_garage_unit IS NULL
+        AND NOT condo.is_question_garage_unit
```

Comment:
Michael helped figure out that we need to make this change in order to return any condos: https://github.com/ccao-data/data-architecture/pull/116#issuecomment-1706939507